server/src/main/java/org/opensearch/index/engine/exec/WriterFileSet.java

```diff
@@ -75,6 +78,16 @@ public String getDirectory() {
         return directory;
     }
 
+    public long getNumRows() {
+        return numRows;
+    }
```

Comment:
We will need to use this num of rows in `CompositeMergePolicy` as well? Can you update that logic as well?